### PR TITLE
[TASK] Commits (https://github.com/SENATOROVAI/intro-cs/issues/5)

### PR DESCRIPTION
**Pull request by nysib**
### Describe your change:

Created python/commits.ipynb explaining each Conventional Commit type,
and made five fictional commits (fix, feat, style, docs, test).

Closes https://github.com/SENATOROVAI/intro-cs/issues/5

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/SENATOROVAI/pre-commit/blob/main/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): Closes https://github.com/SENATOROVAI/intro-cs/issues/5.

### CODE REVIEW

File: python/commits.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Question 5 - Documentation and testing:
+#     Make a fictional commit with the type docs, adding or improving documentation for your new function.
+#
+# git commit -m "docs: add documentation for generateReport function" -m "Added docstring and usage description explaining the function's parameters and output."
+#
+#     Make a fictional commit with the type test, adding tests for that same function.
+#
+# git commit -m "test: add tests for generateReport function" -m "Added unit tests covering normal input, empty input, and edge cases."
+#
+#
```
Comment (nysib) [inline python/commits.py:39]: submitting

**Comment by Toleranzy** [comment]
review done
